src/main/webapp/feed.js

```diff
@@ -0,0 +1,468 @@
+// will be pulled from backend
```

Comment:
I'd probably use a different style notation for variables so they won't be confused with functions.
for example, mockEvent -> mock_event or eventMiddle -> event_middle.

@lazau-g  @kavinkarthik92  What do you all think?

Reply:
Generally, we try to keep the code base style consistent (camelCase or snake_case) across all variables and functions. Since we're using JS, which generally uses camelCase, let's use camelCase across this project.